datatypes/src/raster/operations/grid_blit.rs

```diff
@@ -70,14 +68,55 @@ where
     }
 }
 
-impl<OD, T> GridBlit<OD, T> for Grid3D<T>
+impl<D, T> GridBlit<NoDataGrid<D, T>, T> for Grid2D<T>
+where
+    D: GridSize<ShapeArray = [usize; 2]>
+        + GridBounds<IndexArray = [isize; 2]>
+        + GridSpaceToLinearSpace<IndexArray = [isize; 2]>,
+    T: Pixel + Sized,
+{
+    fn grid_blit_from(&mut self, other: NoDataGrid<D, T>) -> Result<()> {
+        let other_offset_dim = other.bounding_box();
+        let offset_dim = self.bounding_box();
+        let intersection: Option<GridBoundingBox<[isize; 2]>> =
+            offset_dim.intersection(&other_offset_dim);
+        if let Some(intersection_offset_dim) = intersection {
+            let GridIdx([overlap_y_start, overlap_x_start]) = intersection_offset_dim.min_index();
+            let [overlap_y_size, overlap_x_size] = intersection_offset_dim.axis_size();
+
+            for y in overlap_y_start..overlap_y_start + overlap_y_size as isize {
+                for x in overlap_x_start..overlap_x_start + overlap_x_size as isize {
+                    self.set_at_grid_index_unchecked(
+                        [y, x],
+                        other.get_at_grid_index_unchecked([y, x]),
+                    )
+                }
+            }
+        }
+        Ok(())
+    }
```

Comment:
It seems this method cannot fail, right?

Reply:
we use the unchecked get / set methods since we checked the bounds earlier. So i guess it should not fail